tests/client/clientService.test.ts

```diff
@@ -0,0 +1,2274 @@
+import type { Host, Port, TLSConfig } from '@/network/types';
```

Comment:
This file can also be removed, seeing as @scottmmorris has separated this in master now.